src/components/HightlightCaseStudyDetails.tsx

```diff
@@ -1,56 +1,62 @@
 import React from "react";
 import ArrowDown from "./icons/ArrowDown";
+import { Hightlights } from "@/data/case-studies";
 
 interface Props {
-  levels: string[];
-  technologies: string[];
+  setHightlightedElement: (element: string) => void;
 }
 
-const buttonBlock = (entries: string[], title: string, style: string) => (
-  <div>
-    <h1 className="text-center text-gray-500 text-xl font-bold mb-3">
-      {title}
-    </h1>
-
-    <div className="flex flex-col gap-2 p-4 h-72 overflow-y-auto">
-      {entries.map((element, index) => (
-        <button
-          key={index}
-          className={`border border-gray-500 p-4 rounded-lg ${style}`}
-        >
-          {element}
-        </button>
-      ))}
-    </div>
+const HightlightsBotton = (
+  entries: string[],
+  style: string,
+  getCaseStudyDetails: (element: string) => void
+) => (
+  <div className="flex flex-col gap-2 p-4 h-72 overflow-y-auto">
+    {entries.map((element, index) => (
+      <button
+        key={index}
+        onClick={() => getCaseStudyDetails(element)}
+        className={`border border-gray-500 p-4 rounded-lg ${style}`}
+      >
+        {element}
+      </button>
+    ))}
   </div>
 );
 
-const HightlightCaseStudyDetails = ({ levels, technologies }: Props) => {
+const HightlightCaseStudyDetails = ({ setHightlightedElement }: Props) => {
+  const getCaseStudyDetails = (element: string) => {
+    setHightlightedElement(element);
+  };
```

Comment:
You can remove this  function and use setHigtligtedElement only

Reply:
removed